src/tests/functional/plugin/shared/src/behavior/ov_infer_request/io_tensor.cpp

```diff
@@ -170,6 +170,26 @@ TEST_P(OVInferRequestIOTensorTest, canInferWithGetIn) {
     OV_ASSERT_NO_THROW(req.get_tensor(output));
 }
 
+TEST_P(OVInferRequestIOTensorTest, canInferAfterIOBlobReallocation) {
+    ov::Tensor input_tensor, output_tensor;
+    auto in_shape = input.get_shape();
+    auto out_shape = output.get_shape();
+
+    // imitates blob reallocation
+    OV_ASSERT_NO_THROW(input_tensor = req.get_tensor(input));
+    OV_ASSERT_NO_THROW(input_tensor.set_shape({5, 5, 5, 5}));
```

Comment:
As I can see for `OVInferRequestIOTensorTest` tests we create `makeConvPoolReluNoReshapes` with input shapes `{1, 1, 32, 32}` and when we set shape `{5, 5, 5, 5}` blob is not reallocated. 
So let's just increment one of `in_shape` dimension. And set updated shape instead `{5, 5, 5, 5}`.

After update please check that test passed without your changes in `infer_requset`

Reply:
ok